docs/CONFIGURATION.md

```diff
@@ -0,0 +1,42 @@
+# Configuration options
+
+It's possible to configure some settings by passing `--config` argument to the generator. All
+configuration keys are defined in `bindings.cpp` section.
+
+```bash
+uniffi-bindgen-cpp path/to/definitions.udl --config path/to/uniffi.toml
+```
+
+- `cdylib_name` - override the dynamic library name linked by generated bindings, excluding `lib`
```

Comment:
Does `cdylib_name` actually do anything?

Reply:
It is stored and passed to the `BindingsConfig` implementation from uniffi, but otherwise it's not used internally

Reply:
So it's used by uniffi if you're not using the `--library` option

Reply:
After looking at it again, looks like i was wrong, guess this was left over from the start of the project and is in fact no being used at all, removed it from the code and the docs

Reply:
Its used by Kotlin/C# to specify dynamic library name directly in the bindings.